jest/private/noop.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
```

Comment:
I think this is one of those cases that you can simplify to just a two liner sh script
```
#!/usr/bin/env sh
exit 0
```